tensordict/nn/common.py

```diff
@@ -517,6 +517,8 @@ def select_out_keys(self, *out_keys):
                 is_shared=False)
 
         """
+        if len(out_keys) == 1:
```

Comment:
I don't think we need to support this: the function signature and docstrings are pretty clear that a sequence of keys is expected, not a list/tuple of keys. It becomes confusing IMO since we support nested keys. 
Eg
```
TensorDict({
 "a": {"b": foo},
 "b": bar}, [])
```
What is `["a", "b"]` in this context?
To me it should only be the nested entry (`"foo"`)